### PR TITLE
Re-enable server GC & lengthen timeouts

### PR DESCRIPTION
**Pull request by wtgodbe**

### CODE REVIEW

**Comment by wtgodbe** [comment]
@mmitche I saw you merged a change to increase timeouts for non-windows builds (https://github.com/dotnet/coreclr/pull/2984). It looks like you missed changing the timeouts for the flow jobs, so I made that change here. Also @jkotas this would mean we can keep Server GC (which might not even be the problem anyways), so I re-enabled that as well.

**Comment by mmitche** [comment]
Timeouts shouldn't matter for the flow jobs.  Are you seeing an issue?

Sent from my Windows Phone
________________________________

**Comment by wtgodbe** [comment]
@mmitche Yes, for example: http://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/checked_ubuntu_tst_prtest/150/consoleFull#-16758747638101fac1-935e-4556-95f9-7a0e25ca0b52

This is from friday, and timed out after 120 minutes.

**Comment by mmitche** [comment]
@wtgodbe That's the test run, not the flow run.  The flow job isn't timed.

**Comment by mmitche** [comment]
@wtgodbe It should be that the call on 902 should reset the timeout to 240, but it actually didn't.  I'm trying to figure out why.

**Comment by mmitche** [comment]
@wtgodbe Appears it was because the function setting the timeout was ignoring the actual parameter setting the timeout.  FIxing.

File: netci.groovy [hunk 1]
```diff
@@ -929,10 +928,7 @@ build(params + [CORECLR_BUILD: coreclrBuildJob.build.number,
                     }
 
                     Utilities.standardJobSetup(newFlowJob, project, isPR, getFullBranchName(branchName))
-                    //Pri 1 tests need longer timeout
-                    if (scenario == 'pri1') {
-                        Utilities.setJobTimeout(newFlowJob, 240)
-                    }
+                    Utilities.setJobTimeout(newFlowJob, 240)
```
Comment (mmitche) [inline netci.groovy]: You can remove this line.

**Comment by wtgodbe** [comment]
@dotnet-bot test CI please

**Pull request merged**